package.json

```diff
@@ -144,7 +146,8 @@
     "test-style-spec": "cd src/style-spec && npm test",
     "prepublishOnly": "run-s build-flow-types build-dev build-prod-min build-prod build-csp build-css build-style-spec",
     "print-release-url": "node build/print-release-url.js",
-    "codegen": "build/run-node build/generate-style-code.js && build/run-node build/generate-struct-arrays.js"
+    "codegen": "build/run-node build/generate-style-code.js && build/run-node build/generate-struct-arrays.js",
+    "postinstall": "patch-package"
```

Comment:
As discussed in chat, this may affect downstream users of `mapbox-gl` package, so let's do the patch as a part of CI rather than `install` in general.

Reply:
I've dropped `postinstall` after the `testem` update